<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_SkillInfo</class>
 <widget class="QWidget" name="Win_SkillInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="skillImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>30</width>
     <height>200</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>30</width>
     <height>200</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>255</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Coût</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="skillPointsCost">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>300</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-image:url(:/graphicItems/lock.png);
	color:#3c3c3c;
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUnlock">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(250, 250, 250, 100);
	color:#3c3c3c;
}
QPushButton::hover{
	background-color:rgba(250, 250, 250, 180);
	color:#3c3c3c;
}</string>
   </property>
   <property name="text">
    <string>Débloquer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(250, 250, 250, 200);
	color:#3c3c3c;
	/*border:1px solid #4c4c4c;
	border-radius:5px;*/
}
QPushButton::hover{
	background-color:rgba(250, 250, 250, 230);
	color:#3c3c3c;
	/*border:1px solid #4c4c4c;
	border-radius:5px;*/
}</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="skillAddInfo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Matura MT Script Capitals</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:#FFB75C;
}</string>
   </property>
   <property name="text">
    <string>Nécéssite l'apprentissage de la compétence
&quot;Apprenti mage&quot;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="skillName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Matura MT Script Capitals</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:#FFFFFF;
	background-color:rgba(250,250,250,20);
}</string>
   </property>
   <property name="text">
    <string>Corruption</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="spellManaCost">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>HP Simplified</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:#FFFFFF;
	background-color:rgba(250,250,250,20);
	padding-left:5px;
}</string>
        </property>
        <property name="text">
         <string>30</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="skillResume">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:#FFFFFF;
	background-color:rgba(250,250,250,20);
	padding-left:5px;
}</string>
        </property>
        <property name="text">
         <string>Accorde un pourcentage de chance de frapper plus fort en combat</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelManaCost_2">
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>175</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:#FFFFFF;
	background-color:rgba(250,250,250,20);
	padding-right:5px;
}</string>
        </property>
        <property name="text">
         <string>description :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ManaCost_logo">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>HP Simplified</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	border-image:url(:/graphicItems/manaField.png);
	color:#3c3c3c;
	background-color:rgba(250,250,250,20);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelManaCost">
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>135</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MV Boli</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color:#FFFFFF;
	background-color:rgba(250,250,250,20);
	padding-right:5px;
}</string>
          </property>
          <property name="text">
           <string>Coût en mana :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>line_4</zorder>
  <zorder>line_3</zorder>
  <zorder>skillImage</zorder>
  <zorder>label_3</zorder>
  <zorder>skillPointsCost</zorder>
  <zorder>buttonUnlock</zorder>
  <zorder>buttonBack</zorder>
  <zorder>skillAddInfo</zorder>
  <zorder>skillName</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
